src/screens/TransactionDetail.js

```diff
@@ -94,6 +94,25 @@ class TransactionDetail extends React.Component {
 
     try {
       const data = await this.props.wallet.getFullTxById(this.props.match.params.id);
+      for (const output of data.tx.outputs) {
```

Comment:
Should we have unit tests for this?

As a suggestion, it might be a good idea to avoid adding too much business logic here so we have a clearer separation between presentation layer components and business logic.

It could also facilitate unit tests for the business logic, since by putting it here you would have to mount the component and all its dependencies in order to test this logic.

Does it make sense?

Reply:
Maybe, this only affects the UI, but it is a pretty important piece of UI on the transaction data.
But I don't think it's in the scope of this PR, since we should think of how to structure the UI and business logic tests with electron.

Reply:
That's ok for me. But if you think this is important, we should create an issue or KTLO, so we don't forget.